### ISSUE TITLE
expect_compound_columns_to_be_unique not yet implemented in v3

### ISSUE THREAD

**Issue by jdimatteo** [fivetran/great_expectations#3095]
**Is your feature request related to a problem? Please describe.**

The [Table of Expectation Implementations By Backend](https://docs.greatexpectations.io/en/latest/reference/core_concepts/expectations/implemented_expectations.html) documents `expect_compound_columns_to_be_unique` as not yet migrated to the modular Expectations API (0.13).

When I attempt to run this expectation with the v3 api with great expectations version 0.13.23, I get an exception traceback like the following:

```
Traceback (most recent call last):
  File "/v3_poc/example.py", line 202, in <module>
    main()
  File "/v3_poc/example.py", line 198, in main
    run_expectations_without_checkpoint(context)
  File "/v3_poc/example.py", line 164, in run_expectations_without_checkpoint
    validation_result = validator.expect_compound_columns_to_be_unique(["id", "catch_phrase"])
  File "/v3_poc/venv/lib/python3.9/site-packages/great_expectations/validator/validator.py", line 176, in __getattr__
    raise AttributeError(
AttributeError: 'Validator'  object has no attribute 'expect_compound_columns_to_be_unique'
```

**Describe the solution you'd like**

I'd like `expect_compound_columns_to_be_unique` to be migrated so it works with the v3 api.

**Additional context**

Assuming no one else is working on this, I could implement this quicker if someone could point me to a similar PR migrating some other expectation to work with the v3 api. Please feel free to assign me to this issue if no one has time to work on this. I'm trying to migrate to the v3 api and this is one issue blocking me.

Others have run into this issue as well, e.g. https://github.com/great-expectations/great_expectations/issues/2828#issuecomment-862397671

**Comment by munichpavel** [comment]
I also raised this issue in the Slack #support channel on 25th of June. Thanks for putting it here!

**Comment by jcampbell** [comment]
Thanks for flagging this. @alexsherstinsky is looking at this issue right now, but current ETA isn't for a few more weeks. The underlying issue is dealing consistently with how multi-column expectations handle skipping rows for nulls in the BatchRequest (v3) api.

**Comment by talagluck** [comment]
Closing as this is resolved by #3488 and #3477
